### PR TITLE
fix: fix Linear & support warmup start lr & support Cosine eta_min

### PR DESCRIPTION
**Pull request by TingquanGao**
Support setting warmup start lr and eta_min in Cosine.
Fix bug that Linear can not decay to end_lr when setting warmup.

### CODE REVIEW

File: ppcls/optimizer/learning_rate.py [hunk 1]
```diff
@@ -78,25 +80,30 @@ def __init__(self,
                  learning_rate,
                  step_each_epoch,
                  epochs,
+                 eta_min=0,
```
Comment (cuicheng01) [inline ppcls/optimizer/learning_rate.py]: 加入的eta_min和warmup_start_lr可以在注释中说明下

Comment (TingquanGao) [reply ppcls/optimizer/learning_rate.py]: done

**Review by cuicheng01 — APPROVED** [review]
